tests/test_dict.py

```diff
@@ -84,3 +86,94 @@ def test_should_check_item_in_dict(self, shared_memory_dict, key, value):
     def test_should_return_dict_keys(self, shared_memory_dict, key, value):
         shared_memory_dict[key] = value
         assert list(shared_memory_dict.keys()) == [key]
+
+    def test_should_warning_about_move_to_end_deprecation(
+        self, shared_memory_dict, key, value
+    ):
+        shared_memory_dict[key] = value
+        deprecation_message = (
+            'The \'move_to_end\' method will be removed in future versions. '
+            'Use pop and reassignment instead.'
+        )
+        with pytest.deprecated_call(match=deprecation_message):
+            shared_memory_dict.move_to_end(key)
+
+    def test_should_warning_about_last_parameter_deprecation_in_popitem(
+        self, shared_memory_dict, key, value
+    ):
+        shared_memory_dict[key] = value
+        deprecation_message = (
```

Comment:
in my humble opinion, I don't think the tests to check for deprecation messages are necessary. They will also be removed in the future, with the removal of the warning. 

nonetheless .. the `pytest.deprecated_call` is pretty neat

Reply:
I think we can remove it when preparing a new release.